core/src/main/scala/org/apache/spark/rapids/tool/benchmarks/BenchmarkArgs.scala

```diff
@@ -0,0 +1,29 @@
+package org.apache.spark.rapids.tool.benchmarks
+
+import org.rogach.scallop.{ScallopConf, ScallopOption}
+
+class BenchmarkArgs(arguments: Seq[String]) extends ScallopConf(arguments) {
+
+    banner("""
+Benchmarker class for running various benchmarks.
+    """)
+
+  val iterations: ScallopOption[Int] = opt[Int](name = "iterations", short = 'i', default = Some(5),
+    descr = "Total iterations to run")
+
+  val warmupIterations: ScallopOption[Int] = opt[Int](name = "warmupIterations", short = 'w' ,
+    default = Some(3), descr = "Number of warmup iterations to run")
```

Comment:
- We should allow users to skip warmupIterations. i.e., if it is set to 0, it means no warmup
- Add more details about what the purpose of warmupIteration mean. Then if it is set to 0, then no warmup iterations.

Reply:
Current implementation takes 0 as warm up iterations. Adding validators to avoid negative input. Updated description